R/functions__eager.R

```diff
@@ -3,7 +3,8 @@
 #' @param l list of DataFrame, or Series, LazyFrame or Expr
```

Comment:
Need to redocument because `l` was replaced by `...`

Reply:
oups